Storage/FirebaseStorageUI/UIImageView+FirebaseStorage.h

```diff
@@ -17,111 +17,162 @@
+                       placeholderImage:(nullable UIImage *)placeholder;
 
 /**
  * Sets the image view's image to an image downloaded from the Firebase Storage reference.
  * Must be invoked on the main queue.
  *
- * @param storageRef  A Firebase Storage reference containing an image.
- * @param placeholder An image to display while the download is in progress.
- * @param completion  A closure to handle events when the image finishes downloading.
+ * @param storageRef      A Firebase Storage reference containing an image.
+ * @param placeholder     An image to display while the download is in progress.
+ * @param completionBlock A closure to handle events when the image finishes downloading.
  *   The closure is not guaranteed to be invoked on the main thread.
- * @return Returns a FIRStorageDownloadTask if a download was created (i.e. image
- *   could not be found in cache).
  */
-- (nullable FIRStorageDownloadTask *)sd_setImageWithStorageReference:(FIRStorageReference *)storageRef
-                                                    placeholderImage:(nullable UIImage *)placeholder
-                                                          completion:(void (^_Nullable)(UIImage *_Nullable,
-                                                                                        NSError *_Nullable,
-                                                                                        SDImageCacheType,
-                                                                                        FIRStorageReference *))completion;
+- (void)sd_setImageWithStorageReference:(FIRStorageReference *)storageRef
+                       placeholderImage:(nullable UIImage *)placeholder
+                             completion:(void (^_Nullable)(UIImage *_Nullable,
+                                                           NSError *_Nullable,
+                                                           SDImageCacheType,
+                                                           FIRStorageReference *))completionBlock;
 
 /**
  * Sets the image view's image to an image downloaded from the Firebase Storage reference.
  * Must be invoked on the main queue.
  *
- * @param storageRef  A Firebase Storage reference containing an image.
- * @param size        The maximum size of the downloaded image. If the downloaded image
+ * @param storageRef      A Firebase Storage reference containing an image.
+ * @param size            The maximum size of the downloaded image. If the downloaded image
  *   exceeds this size, an error will be raised in the completion block.
- * @param placeholder An image to display while the download is in progress.
- * @param completion  A closure to handle events when the image finishes downloading.
+ * @param placeholder     An image to display while the download is in progress.
+ * @param completionBlock A closure to handle events when the image finishes downloading.
  *   The closure is not guaranteed to be invoked on the main thread.
```

Comment:
I think this comment about which thread the closure is invoked on is no longer true. If this is the case, please update the comment (and the similar comments in other methods).

Reply:
> The closure is not guaranteed to be invoked on the main thread.
Emm...Actually, there is typically the main queue. Because now, the logic is handled by `UIView+WebCache`, there is a dispatch_main_async_safe here.
I update the comment, and remove this line ? Or I change this into:
> The closure is guaranteed to be invoked on the main thread.

Reply:
It should be changed into `The closure is guaranteed to be invoked on the main thread.` instead of removed.

Reply:
Done